### PR TITLE
CPKAFKA-2268 Upgrade commons-validator to 1.5.1 to upgrade transitive…

### PR DESCRIPTION
**Pull request by gardnervickers**
… dep commons-collections-3.2.1 jar to 3.2.2

### CODE REVIEW

File: support-metrics-client/pom.xml [hunk 1]
```diff
@@ -46,7 +46,7 @@
     <dependency>
       <groupId>commons-validator</groupId>
       <artifactId>commons-validator</artifactId>
-      <version>1.4.1</version>
```
Comment (ewencp) [inline support-metrics-client/pom.xml]: can we remove the version from this pom and instead rely on `dependencyManagement` from parent poms like the common pom?

Comment (gardnervickers) [reply support-metrics-client/pom.xml]: I'm not sure what you mean, I don't see `dependencyManagement` used in any of the support-metrics repos.

Comment (gardnervickers) [reply support-metrics-client/pom.xml]: Could we make a Jira for this and address it in a follow up PR? We're trying to get a build out to a client with a new version of commons-collection, the transitive dependency.

Comment (gardnervickers) [reply support-metrics-client/pom.xml]: Unless it's an easy fix of course.

Comment (gardnervickers) [reply support-metrics-client/pom.xml]: Ok thank you, I'll file a Jira for this.

Comment (ijuma) [reply support-metrics-client/pom.xml]: @gardnervickers The idea is to define the dependency in the `common` pom and then here you don't have to mention the version at all. And yes, we can do that in a follow-up, it's pretty trivial.

Comment (gardnervickers) [reply support-metrics-client/pom.xml]: Ok great, I made CP-607 to track this.

Comment (gardnervickers) [reply support-metrics-client/pom.xml]: In the mean time if either of you have a moment to approve this and unblock the merge, that would be appreciated 👍

**Pull request merged**